src/components/navbar/account-link.cjsx

```diff
@@ -11,8 +11,10 @@ module.exports = React.createClass
     router: React.PropTypes.func
 
   redirectToAccount: ->
-    TransitionAssistant.checkTransitionStateTo("Account Profile Page").then ->
-      window.open(CurrentUserStore.getProfileUrl())
+    # workaround for 'No way to come back to dashboard after click on Account'
+    window.open(CurrentUserStore.getProfileUrl())
+    #TransitionAssistant.checkTransitionStateTo("Account Profile Page").then ->
```

Comment:
Should we probably give the new window a name that way if they click the account link twice it'll re-use the first window rather than opening a new tab?  Something like `window.open(CurrentUserStore.getProfileUrl(), 'account-profit')` should do that I think?  https://developer.mozilla.org/en-US/docs/Web/API/Window/open

Also, why not remove the commented code for `TransitionAssistant` and `window.location`?  We can always pull it out of git history if we need to review.